recipes/libcroco/meta.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 Peter Williams and collaborators.
+# This file is licensed under a 3-clause BSD license; see LICENSE.txt.
+
+{% set version = '0.6.12' %}
+{% set version_major = version.rsplit('.', 1)[0] %}
+
+package:
+  name: libcroco
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnome.org/pub/gnome/sources/libcroco/{{ version_major }}/libcroco-{{ version }}.tar.xz
+  sha256: ddc4b5546c9fb4280a5017e2707fbd4839034ed1aba5b7d4372212f34f84f860
+
+build:
+  number: 0
+  script_env:
+    - NJOBS
+
+requirements:
+  build:
+    - glib >=2.51
+    - libxml2 >=2.9
+    - pkg-config
+  run:
+    - glib >=2.51
+    - libxml2 >=2.9
+
+tests:
+  commands:
+    - test -f $PREFIX/lib/libcroco-{{ version_major }}.a      # [linux]
+    - test -f $PREFIX/lib/libcroco-{{ version_major }}.dylib  # [osx]
+    - test -f $PREFIX/lib/libcroco-{{ version_major }}.la
```

Comment:
Don't test for the `.la` file. I think it is better not to include those files in our packages.